pkg/apis/policy/validation/validation.go

```diff
@@ -332,6 +339,44 @@ func validatePodSecurityPolicyVolumes(fldPath *field.Path, volumes []policy.FSTy
 	return allErrs
 }
 
+// getAllFSTypesAsSet returns all actual volume types, regardless
+// of feature gates. The special policy.All pseudo type is not included.
+func getAllFSTypesAsSet() sets.String {
```

Comment:
this was hoisted from k8s.io/kubernetes/pkg/security/podsecuritypolicy/util to allow deleting that package